ocdskingfisherarchive/collection.py

```diff
@@ -60,16 +67,46 @@ def get_size_of_data_folder(self):
             universal_newlines=True,
         )
 
-        self.data_size = int(output.split('\t')[0])
-        return self.data_size
+        self._data_size = int(output.split('\t')[0])
+        return self._data_size
+
+    def get_data_files_exist(self):
+        dir_name = self.config.directory_data + '/' + self.source_id + '/' + \
+                   self.data_version.strftime("%Y%m%d_%H%M%S")
+
+        # It must exist and be a directory
+        if not os.path.exists(dir_name) or not os.path.isdir(dir_name):
```

Comment:
`isdir` is enough (things that don't exist aren't directories). This occurs multiple times.

```suggestion
        if not os.path.isdir(dir_name):
```

Reply:
Actually, you can just `return os.path.isdir(dir_name)`. Never do:

```python
if not condition:
    return False

return True
```

It's overly indirect.

Reply:
I structured it if/return/return on the guess we might have some more checks later to go in the middle (ie does the directory actually have some .json files in it, or is it empty?) but easy to change later - all done.